### PR TITLE
CVSL-2157 Adding initial eligibilty tasklist

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -105,10 +105,10 @@ jobs:
           command: npm run int-test
       - store_test_results:
           path: test_results
+      - store_test_results:
+          path: test_results/playwright/junit.xml
       - store_artifacts:
-          path: integration_tests/videos
-      - store_artifacts:
-          path: integration_tests/screenshots
+          path: test_results/playwright/report
```
Comment (andrewrlee) [inline .circleci/config.yml:111]: fixing artifact output and standardising output location

File: README.md [hunk 2]
```diff
@@ -2,56 +2,7 @@
...
-The rename-project.bash script takes a single argument - the name of the project and calculates from it the project description
-It then performs a search and replace and directory renames so the project is ready to be used.
-
-## Ensuring slack notifications are raised correctly
-
-To ensure notifications are routed to the correct slack channels, update the `alerts-slack-channel` and `releases-slack-channel` parameters in `.circle/config.yml` to an appropriate channel.
-
-## Filling in the `productId`
-
-To allow easy identification of an application, the product Id of the overall product should be set in `values.yaml`. The Service Catalogue contains a list of these IDs and is currently in development here https://developer-portal.hmpps.service.justice.gov.uk/products
```
Comment (andrewrlee) [inline README.md:54]: removing template specific docs

File: helm_deploy/values-dev.yaml [hunk 3]
```diff
@@ -16,6 +16,9 @@ generic-service:
     DPS_URL: 'https://dps-dev.prison.service.justice.gov.uk'
     HMPPS_COMPONENT_API_URL: 'https://frontend-components-dev.hmpps.service.justice.gov.uk'
     AFER_API_URL: 'https://assess-for-early-release-api-dev.hmpps.service.justice.gov.uk'
+    
+    # Needs to be updated
+    PHASE_BANNER_LINK: 'https://assess-for-early-release-dev.hmpps.service.justice.gov.uk'
```
Comment (andrewrlee) [inline helm_deploy/values-dev.yaml:21]: I'll add a ticket to update this

File: server/data/assessForEarlyReleaseApiClient.test.ts [hunk 4]
```diff
@@ -2,19 +2,16 @@ import nock from 'nock'
 import config from '../config'
 import { AssessForEarlyReleaseApiClient } from '.'
 import { AssessmentSummary, OffenderSummary } from '../@types/assessForEarlyReleaseApiClientTypes'
-import { createAssessmentSummary, createOffenderSummary } from './__testutils/testObjects'
-import { mockRequest } from '../routes/__testutils/requestTestUtils'
```
Comment (andrewrlee) [inline server/data/assessForEarlyReleaseApiClient.test.ts:6]: removing express stuff from data layer

File: server/middleware/phaseNameSetup.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+/* eslint no-param-reassign: "off" */
+import { Express } from 'express'
+
+export default (app: Express, phaseName: string) => {
+  app.locals.phaseName = phaseName
+  app.locals.phaseNameColour = phaseName === 'PRE-PRODUCTION' ? 'govuk-tag--green' : ''
+}
```
Comment (andrewrlee) [inline server/middleware/phaseNameSetup.ts:7]: adding phase stuff

File: server/services/caseAdminCaseloadService.test.ts [hunk 6]
```diff
@@ -1,44 +1,60 @@
...
-import { createAssessmentSummary, createOffenderSummary } from '../data/__testutils/testObjects'
-import { mockRequest } from '../routes/__testutils/requestTestUtils'
-
-const offenderSummaryList = [createOffenderSummary({})]
-const assessmentSummary = createAssessmentSummary({})
+import { createAssessmentSummary, createOffenderSummary, createInitialChecks } from '../data/__testutils/testObjects'
 
 const AssessForEarlyReleaseApiClientBuilder = jest.fn()
 const assessForEarlyReleaseApiClient = createAssessForEarlyReleaseApiClient()
-const req = mockRequest({})
```
Comment (andrewrlee) [inline server/services/caseAdminCaseloadService.test.ts:11]: removing express from service layer

File: server/utils/nunjucksSetup.ts [hunk 7]
```diff
@@ -61,6 +61,10 @@ export function registerNunjucks(app?: express.Express): Environment {
   njkEnv.addFilter('initialiseName', initialiseName)
   njkEnv.addFilter('assetMap', (url: string) => assetManifest[url] || url)
   njkEnv.addFilter('toIsoDate', toIsoDate)
+  njkEnv.addFilter(
```
Comment (andrewrlee) [inline server/utils/nunjucksSetup.ts:64]: little helper to allow printing objects nicely in templates when developing

File: server/views/layout.njk [hunk 8]
```diff
@@ -29,7 +31,30 @@
...
+      attributes: {
+        role: "complementary"
+      },
+      tag: {
+        text: phaseName,
+        classes: phaseNameColour
+      },
+      html: 'This is a new service – your <a class="govuk-link" rel="noreferrer noopener" target="_blank" href="'+ phaseBannerLink +'">feedback</a> will help us to improve it.'
+    }) }}
+  {% if not hideBackLink %}
```
Comment (andrewrlee) [inline server/views/layout.njk:45]: adding simple back link and phase banner

File: server/routes/caseAdmin/initialChecks/tasklist.test.ts [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+  caseAdminCaseloadService.getInitialChecks.mockResolvedValue(initialChecks)
+  mockedDate(new Date(2022, 6, 1))
+})
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+describe('GET', () => {
+  it('should render list of licences for approval', async () => {
```
Comment (justin-gardiner) [inline server/routes/caseAdmin/initialChecks/tasklist.test.ts:25]: don't think it should be licences for approval

**Review by justin-gardiner — APPROVED** [review]
LGTM

**Pull request merged**